index.ts

```diff
@@ -1772,6 +1772,65 @@ const http1Page = async (
   } as unknown as ClientHttp2Session;
 };
 
+const workerPage = (proxyHostnameAndPort: string, state: State) => {
+  return staticResponse(
+    `
+const mapping = ${JSON.stringify(
+      Object.entries(state.config.mapping)
+        .filter(key => key && key.length > 1)
+        .map(([key, value]) => {
+          if (typeof value === "string" && value.startsWith("data:"))
+            return [key, "data:text/plain,..."];
```

Comment:
Placeholder?

Reply:
yes sir, otherwise the service-worker javascript code might content some big chunks of text, and there is nothing to do about them